plugins/router/template/router.go

```diff
@@ -326,16 +345,19 @@ func (r *templateRouter) RemoveRoute(id string, route *routeapi.Route) {
 // AddEndpoints adds new Endpoints for the given id.
 func (r *templateRouter) AddEndpoints(id string, endpoints []Endpoint) {
 	frontend, _ := r.FindServiceUnit(id)
-
-	//only add if it doesn't already exist
-	for _, ep := range endpoints {
-		if _, ok := frontend.EndpointTable[ep.ID]; !ok {
-			newEndpoint := Endpoint{ep.ID, ep.IP, ep.Port}
-			frontend.EndpointTable[ep.ID] = newEndpoint
+	//sort the endpoints
+	eps := EndpointSlice(endpoints)
```

Comment:
Why are the endpoints in a different order than what comes out of the endpoints object?